### PR TITLE
Introduce `FiftyOne.Common.CloudStorage`

### PR DESCRIPTION
**Pull request by drasmart**
### Changes
- Add new lib project.
- Add new unit tests project.
- Add `org-name` to workflow configs.

### Why
- [51d Cloud: Unified Storage Service - requirements](https://docs.google.com/document/d/1LSuWzYwRibSKV-tTMru6w4F0JZNAL3KqNfRSyBrznbs)
  - https://github.com/51Degrees/cloud-dotnet/issues/2
  - https://github.com/51Degrees/Pearl.Usage/issues/16

### Dependent branches
- https://github.com/51Degrees/Pearl.Usage/tree/feature/s3-compat
- https://github.com/51Degrees/cloud-dotnet/tree/feature/s3-compat

### CODE REVIEW

**Comment by justadreamer** [comment]
Let's make sure we have commented the public interfaces / classes and add a description for the NuGet package separate from `Fiftyone.Common`.

File: FiftyOne.Common.CloudStorage/FiftyOne.Common.CloudStorage.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Include="..\README.md">
+    <None Include="..\README-CloudStorage.md">
```
Comment (justadreamer) [inline FiftyOne.Common.CloudStorage/FiftyOne.Common.CloudStorage.csproj:17]: let's keep the name README.md but move inside FiftyOne.CommonCloudStorage folder

**Comment by justadreamer** [comment]
Hi @ben51degrees all documentation changes have been made.  The PR is ready for review

**Review by ben51degrees — APPROVED** [review]

**Comment by ben51degrees** [comment]
> Hi @ben51degrees all documentation changes have been made. The PR is ready for review

@justadreamer , I have now approved these changes

**Pull request merged**